test-src/org/pentaho/reporting/platform/plugin/cache/FileSystemCacheBackendTest.java

```diff
@@ -135,7 +148,7 @@ public String getValue() {
 
     private void writeObject( final ObjectOutputStream oos ) throws IOException {
       try {
-        Thread.sleep( 3000L );
+        Thread.sleep( 1000L );
```

Comment:
There is no need to do a thread-sleep. If you want to enforce a certain order of operations, do so via latches. This way you wont slow down the unit tests.